app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Users
+  class RegistrationsController < Devise::RegistrationsController
+    def create
+      @user = User.new(user_params)
+
+      if @user.save
+        redirect_to root_path
+      else
+        render :new, status: :unprocessable_entity
+      end
+    end
+
```

Comment:
add
`private` here
user_params should better be in private section, it's not a controller action + encapsulation